resource-recorder/src/type/mod.rs

```diff
@@ -0,0 +1,100 @@
+use std::{borrow::Cow, fmt::Display, str::FromStr};
+
+use sqlx::Database;
+
+pub mod database;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Type {
+    Database(database::Type),
+    Secrets,
+    StaticFolder,
+    Persist,
```

Comment:
I am just wondering if we should have `Turso` here as well. Potentially the custom resource once we merge the #1082 . What do you think?

Reply:
Yip, I'll have to add `Turso`